### PR TITLE
Security fixes for kiwigrid sidecar image

### PR DESCRIPTION
**Pull request by hisarbalik**

### CODE REVIEW

File: kiwigrid-sidecar/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+FROM python:3.9.0b3-alpine3.12
+
+ENV         PYTHONUNBUFFERED=1
+WORKDIR     /app
+RUN echo "kubernetes==10.0.0" >> requirements.txt
+RUN echo "requests==2.22.0" >> requirements.txt
+
+COPY        --from=builder /app/* /app/
+RUN         apk add --no-cache gcc && \
```
Comment (a-thaler) [inline kiwigrid-sidecar/Dockerfile:11]: formatting is strange, please make it consistent

File: kiwigrid-sidecar/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM kiwigrid/k8s-sidecar:0.1.157 as builder
+
+FROM python:3.9.0b3-alpine3.12
+
+ENV         PYTHONUNBUFFERED=1
+WORKDIR     /app
+RUN echo "kubernetes==10.0.0" >> requirements.txt
```
Comment (a-thaler) [inline kiwigrid-sidecar/Dockerfile:7]: why not copying the requirements from the original image as well? It might change over time

Comment (a-thaler) [reply kiwigrid-sidecar/Dockerfile:7]: I see, it got removed from the image :(

Comment (a-thaler) [reply kiwigrid-sidecar/Dockerfile:7]: maybe let's download it from the repo directly?

Comment (hisarbalik) [reply kiwigrid-sidecar/Dockerfile:7]: > maybe let's download it from the repo directly?

used version 0.1.157 is not available as release download, latest release was 0.1.144

**Pull request merged**